### PR TITLE
Updates May 2021

### PR DESCRIPTION
**Pull request by radinamatic**

### CODE REVIEW

File: docs/manage/options_ini.rst [hunk 1]
```diff
@@ -9,9 +9,11 @@ Installing Kolibri generates a default ``options.ini`` file with all the section
    
 .. _port:
 
+Manage Server Ports
```
Comment (indirectlylit) [inline docs/manage/options_ini.rst:12]:
```suggestion
Manage server ports
```

File: docs/manage/command_line.rst [hunk 2]
```diff
@@ -124,6 +124,19 @@ You can set the specific order for channels in the |learn| **Learn** page accord
     1         a9b25ac9814742c883ce1b0579448337        TESSA - Teacher Resources
     2         95a52b386f2c485cb97dd60901674a98        CK-12 Testing
 
+.. _delete_channels:
+
+
+Delete Channels
```
Comment (indirectlylit) [inline docs/manage/command_line.rst:148]:
```suggestion
Delete channels
```

**Review by indirectlylit — COMMENTED** [review]
Should these be sentence-case?

File: docs/manage/options_ini.rst [hunk 3]
```diff
@@ -38,6 +38,7 @@ If you need Kolibri to serve content resources from a different port than the on
      ZIP_HTTP_PORT = 1234 
      # Substitute 1234 with your desired port number
 
+.. warning:: If you experience problems on
```
Comment (indirectlylit) [inline docs/manage/options_ini.rst:41]: @radinamatic this change looks to be a mistake

image

Comment (radinamatic) [reply docs/manage/options_ini.rst:41]: Ugh, thank you for catching that... 🙏🏽 
That happens when I leave PR hanging for weeks 🤦🏽‍♀️, will correct asap!

**Pull request merged**